requirements.txt

```diff
@@ -1,5 +1,6 @@
 atomicwrites==1.3.0
 attrs==19.3.0
+avro-python3==1.9.1
```

Comment:
why do we need this over fastavro?

Reply:
it's the one that we use in DM -- not sure if there's any logic behind the specific choice

Reply:
the api only uses fastavro - can we try to only use that?

the server should also use only 1 (fastavro) but we should clean that up later